calcom/cal.com
# fix: platform seed script

## What does this PR do?



- Fixes platform seed script not working



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



Run `yarn dx` command and then login using the following credentials - email: "platform@example.com" and password: "PLATFORMadmin2024!". Once you navigate to `/settings/platform` you should have a platform user set up along with a ready to be used OAuth client.
